django/contrib/staticfiles/management/commands/collectstatic.py

```diff
@@ -218,7 +218,15 @@ def clear_dir(self, path):
                          smart_text(fpath), level=1)
             else:
                 self.log("Deleting '%s'" % smart_text(fpath), level=1)
-                self.storage.delete(fpath)
+                full_path = self.storage.path(fpath)
+                if not os.path.exists(full_path) and os.path.lexists(full_path):
+                    # Delete if the file is a broken symlink (see #25346)
+                    try:
+                        os.unlink(full_path)
+                    except OSError as e:
+                        CommandError(e)
```

Comment:
I mean to write `raise CommandError(e)`.

However, [as documented](https://docs.python.org/3/library/os.html#os.remove), `os.unlink` (which is identical to `os.remove`) raises `OSError` when `path` is a directory.
I misunderstood this specification, so I will delete the `try` statement.